pkg/generic/thrift/http.go

```diff
@@ -20,33 +20,60 @@ import (
 	"context"
 
 	"github.com/apache/thrift/lib/go/thrift"
+	"github.com/cloudwego/dynamicgo/conv"
+	"github.com/cloudwego/dynamicgo/conv/t2j"
+	dthrift "github.com/cloudwego/dynamicgo/thrift"
 
 	"github.com/cloudwego/kitex/pkg/generic/descriptor"
+	"github.com/cloudwego/kitex/pkg/protocol/bthrift"
+	"github.com/cloudwego/kitex/pkg/remote"
+	"github.com/cloudwego/kitex/pkg/remote/codec/perrors"
+	cthrift "github.com/cloudwego/kitex/pkg/remote/codec/thrift"
 )
 
 // WriteHTTPRequest implement of MessageWriter
 type WriteHTTPRequest struct {
-	svc              *descriptor.ServiceDescriptor
-	binaryWithBase64 bool
+	svc                             *descriptor.ServiceDescriptor
+	dynamicgoTy                     *dthrift.TypeDescriptor
+	binaryWithBase64                bool
+	dynamicgoConvOpts               *conv.Options // conversion options for dynamicgo
+	dynamicgoConvOptsWithThriftBase *conv.Options // conversion options for dynamicgo with EnableThriftBase turned on
+	hasRequestBase                  bool
+	dynamicgoEnabled                bool
```

Comment:
the 3rd occurence of `dynamicgoEnabled` 😂

Reply:
this as well as the previous comment